src/Mobs/Monster.cpp

```diff
@@ -1176,33 +1179,44 @@ std::unique_ptr<cMonster> cMonster::NewMonsterFromType(eMonsterType a_MobType)
 
 			return cpp14::make_unique<cHorse>(HorseType, HorseColor, HorseStyle, HorseTameTimes);
 		}
+		case mtZombieVillager:
+		{
+			int Profession = Random.RandInt(6);
```

Comment:
#MagicNumber. Use or introduce some constant, which holds the number of `eVillagerType` members.